ui/imports/shared/popups/SendModal.qml

```diff
@@ -105,6 +103,9 @@ StatusDialog {
         property var preferredChainIds: []
         property bool sendTxError: false
         property string sendTxErrorString
+        property string totalTimeEstimate
+        property string totalFeesInEth
+        property string totalFeesInFiat
```

Comment:
Why are we storing these as strings when these are clearly numbers? It prevents us from doing proper formatting :/